### ISSUE TITLE
missing a11y metadata not necessarily required

### ISSUE THREAD

**Issue by mattgarrish** [daisy/ace-gui#117]
The list of missing accessibility metadata provided after the metadata table doesn't provide any indication of whether the metadata is actually required or not. 

I could see people thinking that because `certifierReport` is listed as missing, for example, that means it has to be included, when it's just an optional field.

It might be useful to add whether the missing properties are required, recommended or optional, perhaps in parentheses after the name.

**Comment by danielweck** [comment]
This is not specific to Ace App, I think the HTML report also needs to better present this information. In Ace App there is an indication of “severity” which is directly correlated with recommended / required, but I agree that this is unclear and should be improved.
